source/includes/_10-one-time-sync-company-custom-check.html.md.erb

```diff
@@ -0,0 +1,587 @@
+      <td>No</td>
+      <td>
+        The country of incorporation.
+      </td>
+    </tr>
+    <tr>
+      <td><code>state_of_incorporation</code></td>
+      <td><code>String</code></td>
+      <td>No</td>
+      <td>
+        The state of incorporation.
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+
+### Response fields
+
+<table>
+  <thead>
+    <th>Field</th>
+    <th>Type</th>
+    <th>Required?</th>
+    <th>Description</th>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code>check_output</code></td>
+      <td>Object structured per <code>check_output</code> below</td>
+      <td>If check did not error</td>
+      <td>
+        Indicates the result data for this check. See the
+        <code>check_output</code> section for the full structure of
+        this check.
+      </td>
+    </tr>
+    <tr>
+      <td><code>errors</code></td>
+      <td>
+        An array containing one or more
+        <a href="#check-errors">Errors</a>
+      </td>
+      <td>No</td>
+      <td>
+        A list of errors that occurred while running this check. If this is
+        provided and it's not empty, the result of this check will be
+        considered to be errored by PassFort, regardless of the other contents
+        of this response.
+      </td>
+    </tr>
+    <tr>
+      <td><code>warnings</code></td>
+      <td>
+        An array containing one or more warnings
+      </td>
+      <td>No</td>
+      <td>
+        Essentially the same as <code>errors</code>, except the
+        <code>sub_type</code> and <code>data</code> fields are not present, and
+        providing warnings will not cause PassFort to consider the check
+        as errored.
+      </td>
+    </tr>
+    <tr>
+      <td><code>provider_data</code></td>
+      <td>
+        Any valid JSON value
+      </td>
+      <td>No</td>
+      <td>
+        This should be the structured JSON data returned by the data provider,
+        or a conversion of the data provider's response, to allow
+        PassFort to investigate issues that may arise with your integration.
+      </td>
+    </tr>
+    <tr>
+      <td><code>charges</code></td>
+      <td>
+        An array containing one or more
+        <a href="#check-charges">Charges</a>
+      </td>
+      <td>No</td>
+      <td>
+        A list of charges incurred while running this check. Charges may only
+        be returned if both of these conditions are met:
+        <ul>
+          <li>No errors occurred (i.e. a result was generated).</li>
+          <li>
+            The check is being resold (the commercial relationship is
+            with PassFort).
+          </li>
+        </ul>
+        
+        For demo checks, charges may be returned, but they will not be deducted
+        from the institution's credit.
+        
+        If charges are returned, the total amount charged must be no greater
+        than the maximum cost-per-check specified in the integration's
+        configuration.
+      </td>
+    </tr>
```

Comment:
```suggestion
```
Given we are not supporting reselling for this check type, we should just drop this piece.